### PR TITLE
Change tag from jumbotron to card

### PR DESCRIPTION
**Pull request by chloesw**
I changed jumbotron to cards for cleaner UI.
Please take a look.

### CODE REVIEW

File: research/css/index.css [hunk 1]
```diff
@@ -78,7 +55,7 @@ body {
   background-color: #fff;
 }
 
-.jumbotron img {
+.card img {
```
Comment (harrydrippin) [inline research/css/index.css]: Same as above.

File: research/index.html [hunk 2]
```diff
@@ -24,40 +24,47 @@
         <nav>
           <ul class="nav nav-pills float-right">
             <li class="nav-item">
-              <img src="image/quadcore_logo.png" class="nav-link active" href="#"><span class="sr-only"> ()
```
Comment (harrydrippin) [inline research/index.html]: Only one `nav-link` item should active. Two or more active items are not allowed.

File: research/index.html [hunk 3]
```diff
@@ -24,40 +24,47 @@
...
 
      <section id="news">
-      <div class="jumbotron">
-        <h1 class="display-6">Processor flaw exposes 20 years of devices to new attack</h1>
-        <p class="lead">All week, the tech world has been piecing through rumors of a potentially catastrophic flaw in an entire generation of processors — but with all developers subject to a non-disclosure agreement, there were few hard facts to go on.fermentum massa justo sit amet risus.</p>
-        <p><img src="https://cdn.vox-cdn.com/thumbor/W4PO8Jygk63Xc6JIyjKqinSHqWA=/0x0:640x362/920x613/filters:focal(269x130:371x232):format(webp)/cdn.vox-cdn.com/uploads/chorus_image/image/58195665/Screen_Shot_2011-12-16_at_11.08.27_AM.0.jpg"></p>
+      <div class="card">
+        <div class="card-body">
+          <h1 class="card-title">Processor flaw exposes 20 years of devices to new attack</h1>
+          <p class="card-subtitle mb-2 text muted">All week, the tech world has been piecing through rumors of a potentially catastrophic flaw in an entire generation of processors — but with all developers subject to a non-disclosure agreement, there were few hard facts to go on.fermentum massa justo sit amet risus.</p>
```
Comment (harrydrippin) [inline research/index.html]: Merge `text` and `muted` to `text-muted`.

File: research/index.html [hunk 4]
```diff
@@ -24,40 +24,47 @@
...
 
-      <div class="jumbotron">
-        <h1 class="display-6">Microsoft issues emergency Windows update for processor security bugs</h1>
-        <p class="lead">Microsoft is issuing a rare out-of-band security update to supported versions of Windows today. The software update is part of a number of fixes that will protect against a newly-discovered processor bug in Intel, AMD, and ARM chipsets. </p>
-        <p><img src="https://cdn.vox-cdn.com/thumbor/keNcXCZlEEJifDM__KJH-Ms5MfU=/0x0:765x524/920x613/filters:focal(322x201:444x323):format(webp)/cdn.vox-cdn.com/uploads/chorus_image/image/58195111/windowsbluestock.0.jpg"></p>
-      </div>
+      <div class="card">
+          <div class="card-body">
+            <h1 class="card-title">Microsoft issues emergency Windows update for processor security bugs</h1>
+            <p class="card-subtitle mb-2 text muted">Microsoft is issuing a rare out-of-band security update to supported versions of Windows today. The software update is part of a number of fixes that will protect against a newly-discovered processor bug in Intel, AMD, and ARM chipsets.</p>
```
Comment (harrydrippin) [inline research/index.html]: Same as above.

File: research/index.html [hunk 5]
```diff
@@ -24,40 +24,47 @@
...
+      
 
-      <div class="jumbotron">
-        <h1 class="display-6">LG launches pink V30 ahead of CES</h1>
-        <p class="lead">LG’s pre-CES announcements have so far been high on technology and low on frills, but today the company goes the other way by issuing a new color for its flagship V30 smartphone.</p>
-        <p><img src="https://cdn.vox-cdn.com/thumbor/kmPa0HRLLPOr_xCM0Ep8_kNef_o=/0x0:1800x1200/920x613/filters:focal(756x456:1044x744):format(webp)/cdn.vox-cdn.com/uploads/chorus_image/image/58188111/v30_2.0.jpg"></p>
+      <div class="card">
+        <div class="card-body">
+          <h1 class="card-title">LG launches pink V30 ahead of CES</h1>
+          <p class="card-subtitle mb-2 text muted">LG’s pre-CES announcements have so far been high on technology and low on frills, but today the company goes the other way by issuing a new color for its flagship V30 smartphone.</p>
```
Comment (harrydrippin) [inline research/index.html]: Same as above.

File: research/css/index.css [hunk 6]
```diff
@@ -1,18 +1,15 @@
-/* Space out content a bit */
```
Comment (harrydrippin) [inline research/css/index.css:1]: Consider using [liquid layout](http://maxdesign.com.au/articles/liquid/) for organizing all CSS files include this. You can deal with any viewport size with `@media`.

File: research/css/index.css [hunk 7]
```diff
@@ -35,41 +32,21 @@ body {
     margin: 2rem 0;
   }
   
-  /* Main marketing message and sign up button */
-  .jumbotron {
+  .card {
```
Comment (harrydrippin) [inline research/css/index.css]: Modifying in-library class's style directly is not suggested. Try to make another class for your own style, and overlap on your desired tag. Not only these, `header`, `footer`, `container` may be included.

File: research/css/index.css [hunk 8]
```diff
@@ -35,41 +32,21 @@ body {
...
     .header {
       margin-bottom: 2rem;
     }
-    /* Remove the bottom border on the jumbotron for visual effect */
-    .jumbotron {
-      border-bottom: 0;
-    }
   }
 
 .nav-pills .nav-link.active,
```
Comment (harrydrippin) [inline research/css/index.css:52]: Same as above.

**Review by harrydrippin — COMMENTED** [review]
Good! Please check my several reviews.

File: research/index.html (hunk 3 above)
Comment (ethkim) [inline research/index.html]: Bit puzzled why we have hardcoded content here. Are these meant to be mocks before writing the frontend code?

**Review by ethkim — COMMENTED** [review]
Thanks!

File: research/css/index.css (hunk 6 above)
Comment (chloesw) [reply research/css/index.css:1]: ok i'll refer to this. thanks !

File: research/index.html (hunk 3 above)
Comment (chloesw) [reply research/index.html]: Yes, I intend to test.

File: research/index.html (hunk 5 above)
Comment (chloesw) [reply research/index.html]: ok

File: research/index.html [hunk 9]
```diff
@@ -18,57 +18,59 @@
...
+          <p class="card-subtitle mb-2 text-muted">LG’s pre-CES announcements have so far been high on technology and low on frills, but today the company goes the other way by issuing a new color for its flagship V30 smartphone.</p>
+          <p><img src="https://cdn.vox-cdn.com/thumbor/kmPa0HRLLPOr_xCM0Ep8_kNef_o=/0x0:1800x1200/920x613/filters:focal(756x456:1044x744):format(webp)/cdn.vox-cdn.com/uploads/chorus_image/image/58188111/v30_2.0.jpg"></p>
+        </div>
       </div>
      </section>
 
-     
 
-    <footer class="footer">
+    <footer class="footer footer-content-wrapper">
```
Comment (harrydrippin) [inline research/index.html]: Nesting `<div>` for dividing classes is very effective for styling. Check this snippet:
```
<footer class="footer">
    <div class="footer-content-wrapper">
        <!-- Some tags here -->
    </div>
</footer>
```

**Review by harrydrippin — APPROVED** [review]
Looks good! Just more comments for you. 😄

File: research/index.html (hunk 9 above)
Comment (chloesw) [reply research/index.html]: Thank you so much!!!!!!!!  😃

**Pull request merged**